awx/plugins/isolated/awx_isolated_cleanup.py

```diff
@@ -49,7 +49,7 @@ def main():
                 continue
             elif modtime > folder_cutoff:
                 try:
-                    re_match = re.match(r'\/tmp\/ansible_awx_\d+_.+', path)
+                    re_match = re.match(r'\/tmp\/awx_\d+_.+', path)
```

Comment:
https://github.com/ansible/awx/blob/9bc5d915e12034169088642c29cf4adfa83e2e4c/awx/main/isolated/manager.py#L343

update comment?

Reply:
also, I believe this is a relevant bit here:

https://github.com/ansible/awx/blob/9bc5d915e12034169088642c29cf4adfa83e2e4c/awx/main/tasks.py#L759